.t2d.yml

```diff
@@ -12,8 +12,8 @@ addons:
   apt:
     packages:
       - expect-dev # provides unbuffer utility
-      - cups
-      - libcups2-dev
+      - "cups"
+      - "libcups2-dev"
```

Comment:
Note

These are part of the [oca/oca-ci Dockerfile](https://github.com/OCA/oca-ci/blob/9eff1d7235a52bde738b6ed1503e766a38f6dd70/Dockerfile#L71), as it is the only place we can declare *build* dependencies for third party python libs that don't publish manylinux wheels.

*runtime* deb dependencies, OTOH have to be declared in manifests, in a `deb` key of the `external_dependencies` section.

Reply:
@sbidoul it comes from `base_report_to_printer` but that module declares only `pycups` as a dep indeed https://github.com/OCA/report-print-send/blob/14.0/base_report_to_printer/__manifest__.py#L32
Shall I update that?